packages/tasks/src/tasks/text-to-audio/spec/output.json

```diff
@@ -6,12 +6,13 @@
 	"type": "object",
 	"properties": {
 		"audio": {
-			"description": "The generated audio waveform."
+			"description": "The generated audio waveform.",
+			"comment": "type=binary"
```

Comment:
cc @Wauplin @hanouticelina 

I used this trick to mark a parameter as "binary" (which translates, in the generated Typescript code, to declaring it to be of type `Blob`)

Generation code: https://github.com/huggingface/huggingface.js/pull/1116/files#diff-3a9454ed7c359e2e0522703d9acf4559f6b10fdd67236692bdd4422dd9c8f472R97-R182